<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_Lewy" stretch="0,4,1">
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(200, 200, 200);</string>
          </property>
          <property name="text">
           <string>Obrazek oryginalny</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelObrazek">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(227, 227, 227);
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="histogram_2" name="histogramLewy" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(200, 200, 200);</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_Centralny" stretch="0,4,1">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(200, 200, 200);</string>
          </property>
          <property name="text">
           <string>Obrazek po modyfikacji</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelOutputPicture">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(227, 227, 227);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="histogram_2" name="histogramCentralny" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(200, 200, 200);</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_Prawy" stretch="0,4,1">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(200, 200, 200);</string>
          </property>
          <property name="text">
           <string>Obrazek &quot;odłożony&quot; do porównań.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelOutputPicture2">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(227, 227, 227);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="histogram_2" name="histogramPrawy" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(200, 200, 200);</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>932</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuplik">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionOtworzPlik"/>
    <addaction name="actionExportToCSV"/>
    <addaction name="actionWczytajCSV"/>
    <addaction name="actionWczytajCSVdoObrobki"/>
    <addaction name="actionZapiszZmodyfikowanyObrazJako"/>
   </widget>
   <widget class="QMenu" name="menuEdycja">
    <property name="title">
     <string>Edycja</string>
    </property>
    <widget class="QMenu" name="menuCwiczenia3">
     <property name="title">
      <string>Ćwiczenia 3</string>
     </property>
     <addaction name="actionZastosujMaske"/>
     <addaction name="actionLaplacian"/>
     <addaction name="actionFiltracja0stopni"/>
     <addaction name="actionFiltracja90stopni"/>
     <addaction name="actionRoberts"/>
    </widget>
    <widget class="QMenu" name="menu_wiczenia_4">
     <property name="title">
      <string>Ćwiczenia 4</string>
     </property>
     <addaction name="actionDetekcjaSzkieletu"/>
     <addaction name="actionErozja"/>
     <addaction name="actionDylatacja"/>
     <addaction name="actionOtwarcie"/>
     <addaction name="actionZamkniecie"/>
    </widget>
    <widget class="QMenu" name="menu_wiczenia_5">
     <property name="title">
      <string>Ćwiczenia 5</string>
     </property>
     <addaction name="actionSegmentacja_Watershed"/>
    </widget>
    <widget class="QMenu" name="menu_wiczenia_1_2">
     <property name="title">
      <string>Ćwiczenia 1-2</string>
     </property>
     <widget class="QMenu" name="menuArytmetyczne">
      <property name="title">
       <string>Arytmetyczne</string>
      </property>
      <addaction name="actionDodawanieObrazow"/>
      <addaction name="actionOdejmowanieObrazow"/>
      <addaction name="actionOdejmowanieAbs"/>
      <addaction name="actionAND"/>
      <addaction name="actionOR"/>
      <addaction name="actionXOR"/>
     </widget>
     <widget class="QMenu" name="menuRozci_ganie">
      <property name="title">
       <string>Rozciąganie</string>
      </property>
      <addaction name="actionRozciagniecie"/>
      <addaction name="actionRozciaganieZakresWgParametrow"/>
     </widget>
     <widget class="QMenu" name="menuProgowanie_2">
      <property name="title">
       <string>Progowanie</string>
      </property>
      <addaction name="actionProgowanieBinarneOdwrocone"/>
      <addaction name="actionProgowanieBinarne"/>
      <addaction name="actionProgowanieNprogow"/>
      <addaction name="actionBinaryzacjaZParametrami"/>
      <addaction name="actionProgowanieZachowaniePoziomySzarosci"/>
     </widget>
     <addaction name="actionBlur"/>
     <addaction name="actionGaussian"/>
     <addaction name="actionMedianBlur"/>
     <addaction name="actionSciemnianie"/>
     <addaction name="actionRozjasnianie"/>
     <addaction name="actionWyrownanie"/>
     <addaction name="actionInvert"/>
     <addaction name="menuArytmetyczne"/>
     <addaction name="menuRozci_ganie"/>
     <addaction name="menuProgowanie_2"/>
    </widget>
    <addaction name="actionCofnijZmiane"/>
    <addaction name="separator"/>
    <addaction name="menu_wiczenia_1_2"/>
    <addaction name="menuCwiczenia3"/>
    <addaction name="menu_wiczenia_4"/>
    <addaction name="menu_wiczenia_5"/>
   </widget>
   <widget class="QMenu" name="menuHistogram">
    <property name="title">
     <string>Histogram</string>
    </property>
    <addaction name="actionPokazHistogramOryginalu"/>
    <addaction name="actionPokazHistogramZmodyfikowanego"/>
   </widget>
   <widget class="QMenu" name="menuTryb_modyfikacji">
    <property name="title">
     <string>Opcje</string>
    </property>
    <addaction name="actionPowrotDoOryginalu"/>
    <addaction name="actionZapamietajObrazDoPorownania"/>
    <addaction name="actionPokazTabliceEXCEL"/>
   </widget>
   <addaction name="menuplik"/>
   <addaction name="menuEdycja"/>
   <addaction name="menuHistogram"/>
   <addaction name="menuTryb_modyfikacji"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOtworzPlik">
   <property name="text">
    <string>Otwórz</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionInvert">
   <property name="text">
    <string>Invert</string>
   </property>
  </action>
  <action name="actionPokazHistogramOryginalu">
   <property name="text">
    <string>Oryginał</string>
   </property>
  </action>
  <action name="actionPokazHistogramZmodyfikowanego">
   <property name="text">
    <string>Zmodyfikowany</string>
   </property>
   <property name="toolTip">
    <string>Zmodyfikowany histogram</string>
   </property>
  </action>
  <action name="actionPowrotDoOryginalu">
   <property name="text">
    <string>Powrót do oryginału</string>
   </property>
  </action>
  <action name="actionKonwertujNaGrayScale">
   <property name="text">
    <string>Konwertuj na GrayScale</string>
   </property>
  </action>
  <action name="actionProgowanieBinarne">
   <property name="text">
    <string>Progowanie binarne</string>
   </property>
  </action>
  <action name="actionProgowanieNprogow">
   <property name="text">
    <string>Progowanie n-progów</string>
   </property>
  </action>
  <action name="actionWyrownanie">
   <property name="text">
    <string>Wyrównanie</string>
   </property>
  </action>
  <action name="actionRozciagniecie">
   <property name="text">
    <string>Pełny zakres</string>
   </property>
  </action>
  <action name="actionLaplacian">
   <property name="text">
    <string>Laplacian</string>
   </property>
  </action>
  <action name="actionExportToCSV">
   <property name="text">
    <string>Export do CSV</string>
   </property>
  </action>
  <action name="actionWczytajCSV">
   <property name="text">
    <string>Wczytaj CSV</string>
   </property>
  </action>
  <action name="actionProgowanieBinarneOdwrocone">
   <property name="text">
    <string>Progowanie binarne odwrócone</string>
   </property>
  </action>
  <action name="actionBinaryzacjaZParametrami">
   <property name="text">
    <string>Binaryzacja z parametrami</string>
   </property>
  </action>
  <action name="actionProgowanieZachowaniePoziomySzarosci">
   <property name="text">
    <string>Z zachowaniem poziomow szarości</string>
   </property>
  </action>
  <action name="actionRozciaganieZakresWgParametrow">
   <property name="text">
    <string>Zakres wg parametrów</string>
   </property>
  </action>
  <action name="actionZapiszZmodyfikowanyObrazJako">
   <property name="text">
    <string>Zapisz zmodyfikowany obraz jako...</string>
   </property>
  </action>
  <action name="actionRozjasnianie">
   <property name="text">
    <string>Rozjaśnianie</string>
   </property>
  </action>
  <action name="actionDodawanieObrazow">
   <property name="text">
    <string>Dodawanie</string>
   </property>
  </action>
  <action name="actionOdejmowanieObrazow">
   <property name="text">
    <string>Odejmowanie</string>
   </property>
  </action>
  <action name="actionAND">
   <property name="text">
    <string>AND</string>
   </property>
  </action>
  <action name="actionOR">
   <property name="text">
    <string>OR</string>
   </property>
  </action>
  <action name="actionXOR">
   <property name="text">
    <string>XOR</string>
   </property>
  </action>
  <action name="actionBlur">
   <property name="text">
    <string>Blur</string>
   </property>
  </action>
  <action name="actionGaussian">
   <property name="text">
    <string>Gaussian</string>
   </property>
  </action>
  <action name="actionMedianBlur">
   <property name="text">
    <string>MedianBlur</string>
   </property>
  </action>
  <action name="actionOdejmowanieAbs">
   <property name="text">
    <string>Odejmowanie abs</string>
   </property>
  </action>
  <action name="actionCiagla">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ciągła</string>
   </property>
  </action>
  <action name="actionZastosujMaske">
   <property name="text">
    <string>Zastosuj Maskę</string>
   </property>
  </action>
  <action name="actionFiltracja0stopni">
   <property name="text">
    <string>Filtracja 0 stopni</string>
   </property>
  </action>
  <action name="actionFiltracja90stopni">
   <property name="text">
    <string>Filtracja 90 stopni</string>
   </property>
  </action>
  <action name="actionRoberts">
   <property name="text">
    <string>Roberts</string>
   </property>
  </action>
  <action name="actionDetekcjaSzkieletu">
   <property name="text">
    <string>Detekcja szkieletu</string>
   </property>
  </action>
  <action name="actionErozja">
   <property name="text">
    <string>Erozja</string>
   </property>
  </action>
  <action name="actionDylatacja">
   <property name="text">
    <string>Dylatacja</string>
   </property>
  </action>
  <action name="actionOtwarcie">
   <property name="text">
    <string>Otwarcie</string>
   </property>
  </action>
  <action name="actionZamkniecie">
   <property name="text">
    <string>Zamknięcie</string>
   </property>
  </action>
  <action name="actionSegmentacja_Watershed">
   <property name="text">
    <string>Segmentacja - Watershed</string>
   </property>
  </action>
  <action name="actionSciemnianie">
   <property name="text">
    <string>Ściemnianie</string>
   </property>
  </action>
  <action name="actionCofnijZmiane">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cofnij zmianę...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionZapamietajObrazDoPorownania">
   <property name="text">
    <string>Zapamiętaj obraz do porównania</string>
   </property>
  </action>
  <action name="actionPokazTabliceEXCEL">
   <property name="text">
    <string>Pokaż tablicę EXCEL</string>
   </property>
  </action>
  <action name="actionWczytajCSVdoObrobki">
   <property name="text">
    <string>Wczytaj CSV do obróbki</string>
   </property>
  </action>
  <action name="actionArytmetyczne">
   <property name="text">
    <string>Arytmetyczne</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>histogram_2</class>
   <extends>QWidget</extends>
   <header>histogram_2.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>showBla()</slot>
 </slots>
</ui>
